### DISCUSSION TITLE
Which means "auto_register=True"

### DISCUSSION THREAD

**Discussion by felinto-dev** [RobertCraigie/prisma-client-py#329] [Q&A] [answered]
https://prisma-client-py.readthedocs.io/en/stable/
https://prisma-client-py.readthedocs.io/en/stable/reference/client/

```py
async def main() -> None:
    db = Prisma(auto_register=True)
    await db.connect()
```

```py
from prisma import Prisma

db = Prisma()
await db.connect()
```

I look the documentation but I can't find any explanation. Could someone explain me?

**Comment by RobertCraigie** [comment, marked as answer]
Sorry that isn't clear enough I'll try and make it easier to find in the future.

That parameter means that you can query from model classes, for example:

```py
from prisma import Prisma
from prisma.models import User

prisma = Prisma(auto_register=True)
await prisma.connect()

# this line would fail if auto_register was False or omitted
user = await User.prisma().create(
    data={
        'name': 'Robert',
    },
)
```

This is required as some people may want to have multiple `Prisma` instances at once.

https://prisma-client-py.readthedocs.io/en/stable/reference/model-actions/#registering-a-client-instance

**Reply by felinto-dev** [reply]
awesome!
